### PR TITLE
Add audio message support on desktop.

### LINKED ISSUE #122: Add Audio Message Support for compose build

### PR DESCRIPTION
**Pull request by lotusronin**
Fix for #122 
Only tested on linux desktop, uses the VLCJ library which makes us have a dependency on VLC for audio playing but it was the only solution I could find that didn't cause playback of WAV and MP3s to cut short or the speaker to pop when the audio is first loaded. We could also potentially use VLC for video playback for desktop too so this isn't too bad of a situation. Android has its own media playback API that we will need to learn how to use.

### CODE REVIEW

**Comment by lotusronin** [comment]
During meetup we were not able to run this on the pinephone and it appears vlc package on arch at least does not bundle libvlc. Instead of bundling this library ourselves, the new plan is to run vlc using it's rc interface and communicate with this process over stdin and stdout. We will eventually want to look at having vlc open up a socket and commanding it through that instead.

While this new solution fixes audio issues, the lack of a working vlcj library means we will need to rethink our video playback plans.

File: serif_compose/src/jvmMain/kotlin/xyz/room409/serif/serif_compose/UiPlatform.kt [hunk 1]
```diff
@@ -53,28 +55,120 @@ actual object UiPlatform {
...
+             */
+            //Toss out old stdout data
+            while(reader.ready()) {
+                if(reader.read().toChar() == '>')  break
+            }
+
+            //Send status command and wait for it to process
+            writer.write("status\n")
+            writer.flush()
+            Thread.sleep(5)
```
Comment (Limvot) [inline serif_compose/src/jvmMain/kotlin/xyz/room409/serif/serif_compose/UiPlatform.kt]: I'm a bit worried about this sleep, is there a way we can omit it?

**Review by Limvot — CHANGES_REQUESTED** [review]

File: serif_compose/src/jvmMain/kotlin/xyz/room409/serif/serif_compose/UiPlatform.kt (hunk 1 above)
Comment (lotusronin) [reply serif_compose/src/jvmMain/kotlin/xyz/room409/serif/serif_compose/UiPlatform.kt]: Possibly. Currently the communication with the spawned process is done on the main thread so we can't sit forever waiting but we need to give the process some time to come up with a result for that "status" command we send it. We could probably swap to isPlaying take in a callback, do this status command on a separate thread instead.

Comment (lotusronin) [reply serif_compose/src/jvmMain/kotlin/xyz/room409/serif/serif_compose/UiPlatform.kt]: Updated PR

**Review by Limvot — APPROVED** [review]

**Pull request merged**